css/components/Navigation/navigation.css

```diff
@@ -17,3 +27,101 @@
   vertical-align: middle;
   background: var(--COLOR-RED-500);
 }
+
+.navbar {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
```

Comment:
you could have used flex as helper class and could overwrite justify content property as spacebetween in navbar class